### DISCUSSION TITLE
When will the update support Chrome/127 version

### DISCUSSION THREAD

**Discussion by hisay** [apify/fingerprint-suite#308] [Q&A] [answered]
I don't know how to update network.json to include chrome/127 version in the generated UA yet

**Comment by B4nan** [comment, marked as answer]
It might take some time, we update the models automatically based on new data we gather constantly, so first real people need to update. We don't maintain the network by hand.

**Reply by hisay** [reply]
May I ask if there is still a need to update Chrome/127 or Chrome/128 support version simulation?

**Reply by B4nan** [reply]
I don't understand your question (and it feels like you did not understand my answer above if you are asking this again). We don't maintain the model/network manually, those things happen automatically based on the data we gather from real users. We release new versions every month iirc (also automatically).
